ci/pipelines/deploy.yml

```diff
@@ -913,6 +917,17 @@ jobs:
           <<: *push-app-config
           path: artefact
 
+  - name: deploy-carbon-relay-staging
+    serial_groups: [carbon-relay-stg]
+    plan:
+      - get: omnibus
```

Comment:
I think we could make this deploy a bit less often by putting a path filter on `omnibus` - at the moment we're going to deploy a new Carbon Relay for every change in the infra, which is a bit much.

Not a problem for now though.